### PR TITLE
PIN-3510: removed VerificationRenewal

### PR DESCRIPTION
**Pull request by AsterITA**
https://pagopa.atlassian.net/browse/PIN-3510

### CODE REVIEW

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -4300,13 +4300,13 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  /tenants/{tenantId}/attributes/{attributeId}/updateRenewalStrategy:
+  /tenants/{tenantId}/attributes/{attributeId}: #We have to inform the frontend about this change
```
Comment (galales) [inline src/main/resources/interface-specification.yml]: If this detail is not already in the task, can you please add a comment there?
```suggestion
  /tenants/{tenantId}/attributes/{attributeId}:
```

File: src/main/resources/interface-specification.yml [hunk 2]
```diff
@@ -10679,30 +10679,25 @@ components:
...
       type: object
       properties:
-        renewal:
-          $ref: '#/components/schemas/VerificationRenewal'
         expirationDate:
           type: string
           format: date-time
       required:
-        - renewal
+        - expirationDate
```
Comment (galales) [inline src/main/resources/interface-specification.yml]: any reason why this is required?
what should the request be if the use wants to remove the expiration date? or it is not allowed?

Comment (AsterITA) [reply src/main/resources/interface-specification.yml]: You're right, I didn't think about if the user wants to remove it, is this allowed? @beetlecrunch 

Now that I think about it, shouldn't either `expirationDate` or `extensionDate` not be an `Option` in order to verify if the attribute is expired? What happens if both are `None`? Right now we are checking `extensionDate`.

**Review by TonioGela — APPROVED** [review]

File: src/main/resources/interface-specification.yml (hunk 2 above)
Comment (beetlecrunch) [reply src/main/resources/interface-specification.yml]: Removing expirationDate it's an admitted use case, so it must be optional.

**Review by galales — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**